index.php

```diff
@@ -54,7 +54,7 @@
                 <input type='text' id='premier' name='premier'/><br><br>
                 <label for='search'>Who did they play?</label><br>
                 <input type='text' id='runners-up' name='runners-up'/><br><br>
-                <label for='search'>What was the score? (GG.PP (TOTAL) d. (GG.PP (TOTAL))</label><br>
+                <label for='search'>What was the score? (GG.PP (TOTAL) d. GG.PP (TOTAL))</label><br>
```

Comment:
i assume the for is supposed to be `for="score"`

Reply:
each 'for' has been changed to match the input name